.changeset/shaggy-knives-exercise.md

```diff
@@ -0,0 +1,9 @@
+---
+"@apollo/query-planner": patch
```

Comment:
Not sure if it matters too much, but these should probably be changed to minor now that we're putting it in 2.4.0.